examples/example_rn_macos/App.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React from 'react';
+import type {PropsWithChildren} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  useColorScheme,
+  View,
+} from 'react-native';
+
+import {
+  Colors,
+  DebugInstructions,
+  Header,
+  LearnMoreLinks,
+  ReloadInstructions,
+} from 'react-native/Libraries/NewAppScreen';
+
+import PostHog from 'posthog-react-native';
+
+export let posthog: PostHog | undefined;
+
+export const posthogAsync: Promise<PostHog> = PostHog.initAsync(
+  'phc_pQ70jJhZKHRvDIL5ruOErnPy6xiAiWCqlL4ayELj4X8',
```

Comment:
Should we exclude this or change it to an env variable? Would not like the source to be polluted with data if people run the example

Reply:
it's just an example project with a testing org, it's nice to see events coming through, otherwise every time I run the examples, I have to change the apiKey before, rolling back the changes, its working quite well on the other SDKs as well but its only me testing it anyway.